interface/main/onotes/office_comments.php

```diff
@@ -19,18 +19,22 @@
  */
 
 include_once("../../globals.php");
-include_once("$srcdir/onotes.inc");
 include_once("$srcdir/formatting.inc.php");
 
 //display all of the notes for the day, as well as others that are active from previous dates, up to a certain number, $N
 $N = 10;
+
+$oNoteService = new \services\ONoteService();
 ?>
 
 <html>
 <head>
 
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
```

Comment:
When bring in bootstrap style, also need to support the RTL folks by bringing in the bootstrap rtl after it:
``````
<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
<?php if ($_SESSION['language_direction'] == 'rtl') { ?>
    $this->cssFiles[] = '/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css';
<?php } ?>
``````